tests/src/BookmarkModelTest.php

```diff
@@ -4,24 +4,43 @@
 
 class BookmarkModelTest extends Unit {
 
+	public function setUp() {
+		$this->classTester = new Unit(self::getCalledClass());
+	}
+
 	public function testToXml() {
```

Comment:
~~~php
	public function withParams( $params ) {
		$this->subjectParams = $params;
		return $this;
	}


->withParams(array(
  'key' => 'value',
  'foo' => 'bar',
));
~~~